process/process.go

```diff
@@ -31,18 +30,24 @@ const inZoneTrafficScoreWeight, deviationScoreWeight, sliceScoreWeight = 0.4, 0.
 // StartProcessing starts parsing input file, running simulation and
 // generating output file
 func StartProcessing(inputFile string, outputFile string, alg string) error {
-	inputArray, err := parseInput(inputFile)
+
+	// initialize a goroutine to keep reading row data from input file and
+	// putting the converted row data into a queue
```

Comment:
```suggestion
	// initialize a goroutine to keep read row data from input file and
	// put the converted row data into a queue
```

Reply:
I guess you meant 'goroutine to read xx and put' same as comments below. 
changed to 
'''
// initialize a goroutine to read row data from input file and
// put the converted row data into a queue
'''